data/pref.xml

```diff
@@ -142,7 +142,7 @@
   <global>
     <section id="general">
       <option id="language" type="std::string" default="&quot;en&quot;" />
-      <option id="screen_scale" type="int" default="0" />
+      <option id="screen_scale" type="int" default="2" />
```

Comment:
The `screen_scale` is a special case and we're using the default value `0` (probably something to review in the future along with #4606):

https://github.com/aseprite/aseprite/blob/8323a555007e1db9670b098ce4b1b9c5f8b3d7ad/src/app/modules/gui.cpp#L133

We should keep this equal to `0` (and when we reset this value the customization should be removed from the `.ini` and get it from the theme).

Reply:
I'm now loading the screen and UI scale from the theme, but I agree this should get changed since it's a bit confusing to have a value that sometimes gets clamped or changed like that